evap/staff/tests/test_importers.py

```diff
@@ -429,6 +429,18 @@ def test_existing_course_is_not_recreated(self):
             existing_course_evaluation.participants.all(),
         )
 
+    def test_existing_course_not_counted_in_created_courses_count(self):
+        self.create_existing_course()
+
+        success_messages, __, __ = EnrollmentImporter.process(
+            self.default_excel_content, self.semester, self.vote_start_datetime, self.vote_end_date, test_run=False
+        )
+
+        self.assertIn(
+            "Successfully created 22 courses/evaluations",
+            "".join(success_messages),
+        )
```

Comment:
Could this be part of the test above (...is_not_recreated)? Basically, you would only add the assertion then, right? This would then also allow to easily test with `test_run=True` and `test_run=False` (I think we have tests for both cases?)

Reply:
Yep I made it part of the test above.
Currently, there is no test regarding existing courses that has `test_run=True`. Should I add one that only tests the success message?

Reply:
Yes, there should be a test for the importer test run, and it should be asserted that:
* The merge warning is shown (would be fair if we just assert the the warnings between test run and non-test run are identical, if that works)
* The test success message also has the correct counts. Currently, it doesn't, needs to be fixed here: https://github.com/e-valuation/EvaP/blob/01c1bac1498da374fbea11cfebf2d156d643bfc5/evap/staff/importers.py#L598-L602

Reply:
Ok, I made a test solely for checking the messages because it was too much for the ...is_not_created one